### PR TITLE
Adjust routing to explorer in activities and txHistory

### PR DESCRIPTION
**Pull request by Ayagoumi**
Add navigating to the explorer when looking at a transaction.

When the explorer URL is pre-defined, the button opens a new window that points to the transaction
When it's pre-defined, it routes to the suite explorer and points to the transaction.

Screenshot 2023-02-21 at 11 49 35

Screenshot 2023-02-21 at 11 49 29

### CODE REVIEW

File: src/components/SidePanels/TxHistoryRow.vue
Comment (aeddaqqa) [inline src/components/SidePanels/TxHistoryRow.vue]: im not sure about routing to other explorers bc we don't have the same routing system e.g if we have the explorer of avalanche they are using snowtrace for c-chain transactions which means even if you fill the explorer URL with the correct link you will face an issue with the routing bc the exact path for a tx is not '/c-chain/TX-hash' but '/tx/TX-hash' @Noctunus  what we should do ?

**Review by Noctunus — DISMISSED** [review]
LGTM - but I'd prefer if we'd have the approval of Chris and/or Achraf

**Review by aeddaqqa — DISMISSED** [review]

File: src/components/SidePanels/TxHistoryRow.vue
Comment (Chriserus) [reply src/components/SidePanels/TxHistoryRow.vue]: https://eips.ethereum.org/EIPS/eip-3091 this should be a standard, we mentioned it earlier, if it's introduced on our site, it would ease up some stuff. Idk how when it comes to c/p/x chain

Comment (Ayagoumi) [reply src/components/SidePanels/TxHistoryRow.vue]: I agree, will be updating our explorer URLs to use these standards.

Comment (Ayagoumi) [reply src/components/SidePanels/TxHistoryRow.vue]: @Noctunus do we have to support the routing for Avalanche explorer because it's kind of a mess to handle

C-Chain: `https://snowtrace.io/tx/:hash`
P-Chain: `https://subnets.avax.network/p-chain/tx/:hash`
X-Chain: `https://explorer-xp.avax.network/tx/:hash`

Should I create a function that handles these routing for Avalanche, or not?

Comment (Noctunus) [reply src/components/SidePanels/TxHistoryRow.vue]: We decided in a meeting to make it simple: Only link to the explorer if a magellan URL is given in the network, otherwise the links shall be grayed out or hidden.
This has some more implications as for instance the explorer should not work without magellan and could just display a generic "Selected network is not supported by this explorer" (tbd) page, the tx history should not be grabbed as this also relies on the magellan and maybe other places where we need it.
For this PR you could just make it simple and don't display the links if magellan is not given and do all the other implications in another PR to keep them clear and small.

**Review by Chriserus — APPROVED** [review]

**Review by aeddaqqa — APPROVED** [review]
LGTM

**Pull request merged**
